lib/xapi-stdext-date/date.ml

```diff
@@ -30,9 +30,10 @@ let rfc822_to_string x = x
 
 (* ==== ISO8601/RFC3339 ==== *)
 
-type iso8601 = Ptime.t
+type zone = UTC | Local
+type iso8601 = Ptime.t * zone
```

Comment:
I think this could be misleading. `Ptime.t` is by definition in UTC according to the documentation:

> The type for picosecond precision POSIX timestamps in the range [Ptime.min;Ptime.max]. Note that POSIX timestamps, and hence values of this type, are by definition always on the UTC timeline. 

I don't have a good suggestion, though.

Reply:
The zone specifies the preferred way to convert to a string, so perhaps call its `PrintUTC | PrintLocal`.